benchmark/fs/readfile-clogging.js

```diff
@@ -0,0 +1,74 @@
+// Call fs.readFile over and over again really fast.
+// Then see how many times it got called.
+// Also report how many times we were able to run a small concurrent
+// operation on the threadpool.
+'use strict';
+
+const path = require('path');
+const common = require('../common.js');
+const filename = path.resolve(__dirname, '.removeme-benchmark-garbage');
+const fs = require('fs');
+const zlib = require('zlib');
+
+const bench = common.createBenchmark(main, {
+  dur: [5],
+  len: [1024, 16 * 1024 * 1024],
+  concurrent: [1, 10]
+});
+
+function main(conf) {
+  const len = +conf.len;
+  try { fs.unlinkSync(filename); } catch (e) {}
+  var data = Buffer.alloc(len, 'x');
+  fs.writeFileSync(filename, data);
+  data = null;
+
+  var zipData = Buffer.alloc(1024, 'a');
+
+  var reads = 0;
+  var zips = 0;
+  var bench_ended = false;
+  bench.start();
+  setTimeout(function() {
+    console.log(`bench ended, reads ${reads} zips ${zips}`);
+    bench_ended = true;
+    bench.end(reads);
+    bench.end(zips);
+    try { fs.unlinkSync(filename); } catch (e) {}
+    process.exit(0);
```

Comment:
This isn't really safe since `process.send()` used by `bench.end()` is not synchronous. It's better to just return early in `afterRead()` and `afterZip()`  when `bench_ended === true` and let the process exit naturally.

Reply:
OK, this `process.exit(0)` is just a copy/paste from `benchmark/fs/readfile.js`. I'll fix this in both places.